cross/homeassistant/Makefile

```diff
@@ -0,0 +1,43 @@
+NAME = homeassistant
+PKG_NAME = homeassistant
+include version
+PKG_EXT = tar.gz
+PKG_DOWNLOAD_METHOD = git
+PKG_DIST_SITE = https://github.com/home-assistant/home-assistant.git
+PKG_DIST_FILE = $(PKG_NAME)-git$(PKG_GIT_HASH).$(PKG_EXT)
+PKG_DIR = $(PKG_NAME)-git$(PKG_GIT_HASH)
+
+# scaffolding to allow just a basic git clone as a downloaded archive (from cross-cc)
```

Comment:
There is easier way to get git hash. See https://github.com/SynoCommunity/spksrc/blob/master/cross/gateone/Makefile as an example.
By the way, I recommend to stick to git tag 0.80.3